fe/src/main/java/org/apache/doris/load/routineload/RoutineLoadTaskInfo.java

```diff
@@ -53,4 +68,16 @@ public void setLoadStartTimeMs(long loadStartTimeMs) {
     public long getLoadStartTimeMs() {
         return loadStartTimeMs;
     }
+
+    public long getTxnId() {
+        return txnId;
+    }
+
+    abstract RoutineLoadTask createStreamLoadTask(long beId);
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
What if obj is not an instance of  RoutineLoadTaskInfo?
You should check it first